templates/init/functions/typescript/index.ts

```diff
@@ -1,9 +1,19 @@
-import * as functions from "firebase-functions";
+/**
+ * Import function triggers from their respective submodules:
+ *
+ * import { onCall } = from "firebase-functions/v2/https";
+ * import { onDocumentWritten } = from "firebase-functions/v2/firestore";
+ *
+ * See a full list of supported triggers at https://firebase.google.com/docs/functions
+ */
+
+ import { onRequest } from "firebase-functions/v2/https";
```

Comment:
Need to remove spaces inside curly brackets to make ESLint happy. Also do it in the commented out lines at the top